tests/launch-mock-event-recorder-tests.sh

```diff
@@ -31,13 +31,13 @@ export DBUS_SESSION_BUS_PID
 trap finally EXIT
 
 # Start the mock service and add its methods.
-python -m dbusmock --system \
+python3 -m dbusmock --system \
```

Comment:
Sounds good. Given the well-knownness of the name `python3` I am comfortable leaving out the extra level of indirection unless it creates problems for developers.